src/server/engine_shard_set.cc

```diff
@@ -39,8 +49,260 @@ using absl::GetFlag;
 
 namespace {
 
+constexpr DbIndex DEFAULT_DB_INDEX = 0;
+
+enum class ObjectTypeForScan : uint8_t {
```

Comment:
Use this to filter types - may we will not need this but we will have the option?

Reply:
i think lines 54-128 are not needed for this pr